app/assets/javascripts/application/nav-close.js

```diff
@@ -0,0 +1,11 @@
+$(document).click(function(event) {
+  $navmenu = $('#nav-menu');
+  $target = $(event.target);
```

Comment:
These variables are getting global scope - they should be prefixed with `var`.

Reply:
See above.